go/Makefile

```diff
@@ -9,8 +9,6 @@ vet:
 
 lint:
 	golint ./... | grep -v ^vendor | grep -v ^protocol | grep -v comment | grep -v KeysById | grep -v "error should be the last type" | grep -v stutters || echo "Lint-free!"
-	# Strict linting on saltpack
-	golint saltpack/...
```

Comment:
Go for it...I didn't even know/forgot the `test` target was there.  

I just wanted to get this in so that the git commit hook will stop complaining.